config_test.go

```diff
@@ -23,6 +23,8 @@ type buildResult struct {
 }
 
 func TestE2E(t *testing.T) {
+	t.Parallel()
```

Comment:
I believe that we had placed these later b/c the `certstrap` library the `certtest` package uses is not safe for concurrent use: https://github.com/square/certstrap/issues/72

Reply:
We might end up seeing flakiness in CI, with this specific change, we can validate by bumping it in diego-release

Reply:
see: https://github.com/cloudfoundry/tlsconfig/commit/d8e16d7d2c43cc9ebe0d0c15679ecc9c948a7912

Reply:
Ah, I’m sorry. I’ll revert this bit and tidy up this afternoon.

Reply:
Hi @sunjayBhatia @xoebus, I figured this could possibly be problematic, hence it was added as a separate commit. Revert away!